pkg/txm/clientwrappers/dualbroadcast/meta_metrics.go

```diff
@@ -0,0 +1,100 @@
+package dualbroadcast
+
+import (
+	"context"
+	"strconv"
+	"time"
+
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/metric"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/beholder"
+)
+
+// MetaMetrics handles all Meta-related metrics via OTEL
+type MetaMetrics struct {
+	chainID           string
+	statusCodeCounter metric.Int64Counter
+	latencyHistogram  metric.Int64Histogram
+	bidHistogram      metric.Int64Histogram
+	errorCounter      metric.Int64Counter
+}
+
+// NewMetaMetrics creates a new MetaMetrics instance
+func NewMetaMetrics(chainID string) (*MetaMetrics, error) {
+	statusCodeCounter, err := beholder.GetMeter().Int64Counter("meta_endpoint_status_codes")
```

Comment:
Might want to inject the metrics as args instead of using the global hook `beholder.GetMeter()`

Reply:
For testability primarily. This approach mixes all package tests together, and sometimes those tests even run distinct nodes. Not a blocker though